examples/gno.land/r/gov/dao/v3/README.md

```diff
@@ -0,0 +1,185 @@
+# GOVDAO Specifications
+
+## 1. Overview
+**GovDAO** is a governance body with three membership tiers—**T1**, **T2**, and **T3**—along with defined voting power, membership requirements, size targets, and compensation policies. 
+
+---
+
+## 2. Membership Tiers
+
+### 2.1 Tier Definitions
+- [X] **T1 (Core Tier)**  
+  - [X] Highest tier; self-selecting membership with a *supermajority* vote from T1.
+  - [X] Membership can only be withdrawn by supermajority vote *with cause*.
+- [X] **T2**  
+  - [X] Selected by GovDAO with T3 abstaining, requiring a *simple majority* vote.
+  - [X] Membership can be withdrawn for any reason.
```

Comment:
`Membership can be withdrawn` with T3 abstaining?

Reply:
Same clarification as [here](https://github.com/gnolang/gno/pull/3389#discussion_r1996171135) but with T1 + T2 / without T3 only for selecting and not for withdrawal.